forms/enrollment/naccid-enrollment-form_questions_and vars.csv

```diff
@@ -0,0 +1,28 @@
+1,Is this to assign a NACCID for the first time or to transfer a participant?,ENRLTYPE,Always,1-2,Integer,
+2a,Participant's month of birth,ENRLBIRTHMO,Always,Integers 1-12,Integer,"I know this var is >10 characters, but I think I'd prefer them to match the UDSv4 A1 variables rather than meet our made-up character limit. Jessica agrees"
+2b,Participant's year of birth,ENRLBIRTHYR,Always,1850 - current year minus 15,Integer,"I know this var is >10 characters, but I think I'd prefer them to match the UDSv4 A1 variables rather than meet our made-up character limit. Jessica agrees"
+3,How many years of education has the participant completed?,ENRLEDUC,Always,"Integers 0-36, 99",Integer,
+4a,Gender--Man,ENRLGENMAN,Always,1 or blank,Integer,"I know this var is >10 characters, but I think I'd prefer them to match the UDSv4 A1 variables rather than meet our made-up character limit. Jessica agrees"
+4b,Gender--Woman,ENRLGENWOMAN,Always,1 or blank,Integer,"I know this var is >10 characters, but I think I'd prefer them to match the UDSv4 A1 variables rather than meet our made-up character limit. Jessica agrees"
+4c,Gender--Transgender man,ENRLGENTRMAN,Always,1 or blank,Integer,"I know this var is >10 characters, but I think I'd prefer them to match the UDSv4 A1 variables rather than meet our made-up character limit. Jessica agrees"
+4d,Gender--Transgender woman,ENRLGENTRWOMAN,Always,1 or blank,Integer,"I know this var is >10 characters, but I think I'd prefer them to match the UDSv4 A1 variables rather than meet our made-up character limit. Jessica agrees"
+4e,Gender--Non-binary/genderqueer,ENRLGENNONBI,Always,1 or blank,Integer,"I know this var is >10 characters, but I think I'd prefer them to match the UDSv4 A1 variables rather than meet our made-up character limit. Jessica agrees"
+4f,Gender--Two-spirit,ENRLGENTWOSPIR,Always,1 or blank,Integer,"I know this var is >10 characters, but I think I'd prefer them to match the UDSv4 A1 variables rather than meet our made-up character limit. Jessica agrees"
+4g,Gender--I use a different term (specify),ENRLGENOTH,Always,1 or blank,Integer,"I know this var is >10 characters, but I think I'd prefer them to match the UDSv4 A1 variables rather than meet our made-up character limit. Jessica agrees"
+4g1,Gender--I use a different term specify,ENRLGENOTHX,Conditional,text,String,"I know this var is >10 characters, but I think I'd prefer them to match the UDSv4 A1 variables rather than meet our made-up character limit. Jessica agrees"
+4h,Gender--Don't know,ENRLGENDKN,Always,1 or blank,Integer,"I know this var is >10 characters, but I think I'd prefer them to match the UDSv4 A1 variables rather than meet our made-up character limit. Jessica agrees"
+4i,Gender--Prefer not to answer,ENRLGENNOANS,Always,1 or blank,Integer,"I know this var is >10 characters, but I think I'd prefer them to match the UDSv4 A1 variables rather than meet our made-up character limit. Jessica agrees"

```

Comment:
4a - 4i fields are not required but marked as "Always"?

Reply:
At least one of them is required to be checked. This is how we wrote it for UDSv4 as well. We didn't really know how to represent that.

Reply:
Wouldn't it be more appropriate to mark these variables as "Conditional"? The error check enrl-m-025 will capture the condition that at least one of these need to be filled. If we mark them as required fields form will be rejected unless all of them are checked.